pkg/utils/maps/maps_test.go

```diff
@@ -0,0 +1,120 @@
+package maps
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestKeys(t *testing.T) {
+	tests := []struct {
+		name string
+		m    map[string]int
+		want []string
+	}{
+		{
+			name: "Empty map",
+			m:    map[string]int{},
+			want: []string{},
+		},
+		{
+			name: "Single key",
+			m:    map[string]int{"a": 1},
+			want: []string{"a"},
+		},
+		{
+			name: "Multiple keys",
+			m:    map[string]int{"a": 1, "b": 2, "c": 3},
+			want: []string{"a", "b", "c"},
+		},
+	}
+
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			got := Keys(tc.m)
+			if !reflect.DeepEqual(got, tc.want) {
+				t.Errorf("Keys() = %v, want %v", got, tc.want)
+			}
+		})
+	}
+}
+
+func TestValues(t *testing.T) {
+	tests := []struct {
+		name string
+		m    map[string]int
+		want []int
+	}{
+		{
+			name: "Empty map",
+			m:    map[string]int{},
+			want: []int{},
+		},
+		{
+			name: "Single value",
+			m:    map[string]int{"a": 1},
+			want: []int{1},
+		},
+		{
+			name: "Multiple values",
+			m:    map[string]int{"a": 1, "b": 2, "c": 3},
+			want: []int{1, 2, 3},
```

Comment:
The iteration order of a map is not stable in Golang

Reply:
Yeh . will add function sort and compare it

Reply:
Yes